### DISCUSSION TITLE
New Whisper Turbo model

### DISCUSSION THREAD

**Discussion by Narbs1** [Purfview/whisper-standalone-win#307] [Ideas]
Will you update your project for the new v3-turbo model that was released recently? That would be awesome.

**Comment by Moogajube** [comment]
I am using it now?  I downloaded the turbo model and copied into _models folder using directory named "faster-whisper-turbo", then ran Faster-Whisper with --model turbo argument

**Reply by Fetterbr61** [reply]
please add the download link or address, i cant find the model

**Reply by misutoneko** [reply]
Ask and thou shalt receive:
https://huggingface.co/Infomaniak-AI/faster-whisper-large-v3-turbo/tree/main

This one worked for me. Note that the model.bin file alone isn't sufficient, you need those other files as well.

EDIT: @1001ruchka, yes all the files were from the link.
Note that in the HF directories the download link is located after the file size.
There's a link in the file name as well, but it's not the same URL!
So if you try to download with that link (via "Save as..", or whatever), you will get a HTML page instead of the actual file.

**Reply by 1001ruchka** [reply]
> Ask and thou shalt receive: https://huggingface.co/Infomaniak-AI/faster-whisper-large-v3-turbo/tree/main
> This one worked for me. Note that the model.bin file alone isn't sufficient, you need those other files as well.

I'm running with an error:
`Traceback (most recent call last):
File "D:\whisper-fast\__main__.py", line 1600, in <module>
File "D:\whisper-fast\__main__.py", line 1431, in cli
File "faster_whisper\transcribe.py", line 140, in __init__
RuntimeError: [json.exception.parse_error.101] parse error at line 1, column 1: syntax error while parsing value - invalid literal; last read: '<'     [18300] Failed to execute script '__main__' due to unhandled exception!     `

Where did you get the json files from? From the link you provided (I did) or did you replace them with your own?

UPDATE:
Replaced the json files from the regular large-v3 model and the large-v3-turbo model worked.

**Reply by Purfview** [reply]
>Replaced the json files from the regular large-v3 model and the large-v3-turbo model worked.

Don't do that.

**Comment by ClaireCJS** [comment]
Does this model have better accuracy than large-v2?

**Reply by misutoneko** [reply]
Not really, no. Based the WER scores I've seen it should be fairly close though (for english).
The main feature of this model is _SPEED_ and it has been accomplished by pruning the v3 model, with some additional finetuning.

**Comment by Purfview** [comment]
Autodownload for the turbo model is added in v193.1
